config/v1/types_feature.go

```diff
@@ -105,7 +105,9 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		Enabled:  []string{},
 		Disabled: []string{},
 	},
-	TechPreviewNoUpgrade: newDefaultFeatures().toFeatures(),
+	TechPreviewNoUpgrade: newDefaultFeatures().with(
+		"ExternalCloudProvider",
```

Comment:
should we have a comment here?
```suggestion
		"ExternalCloudProvider", // sig-cloud-provider, danil-grigorev, elmiko
```